test/manual/testplan.md

```diff
@@ -1,87 +1,200 @@
+| 1  | Update Installation                                           | 1. Install a previous version<br>2. Update to the new version                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                           | :construction: Win<br>:construction: macOS<br>:construction: Linux |                              |
+| 2  | Install the new version                                       | Delete any previous version<br><br> **Windows:** <br> - Install the desktop client using .msi installer. <br> - During installation, check the MSI details page for: <br> &nbsp;&nbsp; - Verified publisher <br> &nbsp;&nbsp; - Certificate expiry <br><br> **Mac:** <br> - Install the desktop client using .pkg installer <br><br> **Ubuntu or Debian with GNOME desktop:** <br> - Run the client as AppImage. <br><br> **Fedora with GNOME desktop:** <br> - Install the client. <br> - Install the Nautilus extension: **All platforms:** <br> - Check the version from the Settings tab -> About | **Windows:** <br>Verified publisher: OpenCloud GmbH <br> Show information about the publisher's certificate: <br> - Issued to: OpenCloud GmbH <br> - Issued by: `<Issuer>` Code Signing CA <br> - Valid from `<start-date>` to `<end-date>` <br><br> **All platforms:** <br> installed version is correct | :construction: Win<br>:construction: macOS<br>:construction: Linux |                              |
```

Comment:
The official release will only be available in the Windows Store, we don't have an msi installler.
Previews and dailys are also availablle as a .7za or an nsis installer. But those are only meant for testig.

Reply:
The linux shell extensions are currently only available on openSUSE.
A smoke test should also be done on a KDE Plasma installation.

Reply:
Windows: I've the MSI installation with EXE installation. I guess we have to use the .exe from the release page because we have to test before the product is pushed to the store. Or do I misunderstand and we should test the store version?

Linux: I've fixed the text, so that always the Appimage is used except for OpenSuse